src/antelope/wallets/authenticators/WalletConnectAuth.ts

```diff
@@ -94,11 +105,57 @@ export class WalletConnectAuth extends EVMAuthenticator {
                 this.trace('login', 'web3Modal.openModal()');
                 const web3Modal = new Web3Modal(this.options, this.wagmiClient);
                 web3Modal.subscribeModal(async (newState) => {
-                    this.trace('login', 'web3Modal.subscribeModal ', newState, localStorage.getItem('wagmi.connected'));
+                    const chainSettings = useChainStore().currentChain.settings as EVMChainSettings;
+                    const wagmiConnected = localStorage.getItem('wagmi.connected');
```

Comment:
Maybe you can put this `const wagmiConnected` at the beginning of the function and use it also in line 101

```typescript
    if (localStorage.getItem('wagmi.connected')) {
```